compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropContainer.skiko.kt

```diff
@@ -83,17 +83,21 @@ internal interface InteropContainer {
  * Counts the number of interop components before the given native view in the container.
  *
  * @param holder The holder for native view to count interop components before.
+ * @param predicate returns true if the given interop holder should be counted.
  * @return The number of interop components before the given native view.
  */
-internal fun InteropContainer.countInteropComponentsBelow(holder: InteropViewHolder): Int {
+internal fun InteropContainer.countInteropComponentsBelow(
+    holder: InteropViewHolder,
+    predicate: InteropContainer.(InteropViewHolder) -> Boolean = { contains(it) }
```

Comment:
can we give this predicate more descriptive name? something like `countIf` or `includeIf` ?